src/pymor/reductors/stokes.py

```diff
@@ -0,0 +1,145 @@
+from pymor.operators.constructions import IdentityOperator
+from pymor.reductors.basic import ProjectionBasedReductor, StationaryLSRBReductor, StationaryRBReductor
+from pymor.vectorarrays.constructions import cat_arrays
+
+
+class StationarySupremizerGalerkinStokesReductor(ProjectionBasedReductor):
+    """Projection-based reductor for the stationary stokes equation.
+
+    Uses supremizer enrichment to stabilize the velocity space, then computes
+    a Galerkin projection of the full order model onto the reduced space.
+
+    Parameters
+    ----------
+    fom
+        The Stokes |SaddlePointModel| to reduce.
+    RB_u
+        The basis of the reduced velocity space onto which to project.
+        If `None`, an empty basis is used.
+    RB_p
+        The basis of the redcued pressure space onto which to project.
+        If `None`, an empty basis is used.
+    u_product
+        Inner product |Operator| w.r.t. which `RB_u` is orthonormalized.
+        If `None`, the Euclidean inner product is used.
+    p_product
+        Inner product |Operator| w.r.t. which `RB_p` is orthonormalized.
+        If `None`, the Euclidean inner product is used.
+    check_orthonormality
+        See :class:`~pymor.reductors.basic.ProjectionBasedReductor`.
+    check_tol
+        See :class:`~pymor.reductors.basic.ProjectionBasedReductor`.
+    """
+
+    def __init__(self, fom, RB_u=None, RB_p=None, u_product=None, p_product=None,
+                 check_orthonormality=None, check_tol=None):
+        assert isinstance(fom, SaddlePointModel)
+        RB_u = fom.solution_space.subspaces[0].empty() if RB_u is None else RB_u
+        RB_p = fom.solution_space.subspaces[1].empty() if RB_p is None else RB_p
+        assert RB_u in fom.solution_space.subspaces[0]
+        assert RB_p in fom.solution_space.subspaces[1]
+
+        self.u_product = u_product
+        self.supremizers = fom.solution_space.subspaces[0].empty()
+
+        super().__init__(fom, {'RB_u': RB_u, 'RB_p': RB_p}, {'RB_u': u_product, 'RB_p': p_product},
+                        check_orthonormality=check_orthonormality, check_tol=check_tol)
+
+    def project_operators(self):
+        fom = self.fom
+        RB_u = self.bases['RB_u']
+        RB_p = self.bases['RB_p']
+
+        if len(self.supremizers) < len(RB_p):
+            # Enrich velocity basis RB_u with one supremizer per pressure basis vector:
+            # len(RB_u) <- len(RB_u) + len(RB_p)
+            with self.logger.block(
+                'Enriching RB_u with supremizers'
+            ):
+                supremizers = self.compute_supremizers(RB_p, offset=len(self.supremizers))
+                RB_u_enriched = cat_arrays([RB_u, self.supremizers, supremizers])
+                self.supremizers = supremizers
+        else:
+            RB_u_enriched = cat_arrays([RB_u, self.supremizers])
+
+        RB_u_enriched = gram_schmidt(RB_u_enriched, offset=len(RB_u), product=self.u_product, copy=False, check=False)
```

Comment:
```suggestion
        RB_u_enriched = gram_schmidt(RB_u_enriched, offset=len(RB_u), product=self.u_product, copy=False)
```
I think, in general we do not disable the orthonormality check unless there is good reason to do so.